go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/ServiceWeaver/weaver-k8s
 go 1.20
```

Comment:
Do you mind changing weaver-k8s to weaver-kube? I forgot about it

Reply:
No worries, done.